frontend/javascripts/oxalis/view/right-border-tabs/nuclei_inferral_modal.js

```diff
@@ -0,0 +1,120 @@
+// @flow
+import React, { useEffect, useState } from "react";
+import { type APIDataset } from "types/api_flow_types";
+import { Modal, Select, Button } from "antd";
+import { startNucleiInferralJob } from "admin/admin_rest_api";
+import { getColorLayers } from "oxalis/model/accessors/dataset_accessor";
+import Toast from "libs/toast";
+import { Unicode } from "oxalis/constants";
+
+const { ThinSpace } = Unicode;
+
+type Props = {
+  dataset: APIDataset,
+  handleClose: () => void,
+};
+
+export default function NucleiInferralModal(props: Props) {
+  const { dataset, handleClose } = props;
+  const [selectedColorLayerName, setSelectedColorLayerName] = useState(null);
+  const colorLayers = getColorLayers(dataset);
+  useEffect(() => {
+    if (colorLayers.length === 1) {
+      setSelectedColorLayerName(colorLayers[0].name);
+    }
+  });
+  if (colorLayers.length < 1) {
+    return null;
+  }
+
+  const onChange = selectedLayerName => {
+    setSelectedColorLayerName(selectedLayerName);
+  };
+
+  const startJob = async () => {
+    if (selectedColorLayerName == null) {
+      return;
+    }
+    try {
+      await startNucleiInferralJob(
+        dataset.owningOrganization,
+        dataset.name,
+        selectedColorLayerName,
+      );
+      Toast.info(
+        <>
```

Comment:
Is there any specific reason why you did not use `React.Fragement` here? @philippotto

Reply:
`<>` is just syntactic sugar for `React.Fragment` :)